content/docs/how-to/best-practices.md

```diff
@@ -0,0 +1,41 @@
+# Best Practices
```

Comment:
what is the plan with this section? now it has more question then answers :)

Reply:
No major plan right now but there's #72 with more info on what to do next. This just begins to address that issue (as a way to remove this content from **What is DVC?**, but further work on it is out of scope, so I'll probably just hide it from the sidebar for this PR.